### PR TITLE
At&t fix

### PR DESCRIPTION
**Pull request by mklauber**

### CODE REVIEW

**Comment by almathew** [comment]
:+1:  @JLMcLaugh

File: import_jobs.py [hunk 1]
```diff
@@ -613,6 +613,15 @@ def add_jobs(jobs, upload_chunk_size=1024):
     """
     conn = Solr(settings.HAYSTACK_CONNECTIONS['default']['URL'])
     num_jobs = len(jobs)
+    
+    # AT&T Showed that large numbers of MOCs can cause import issues due to the size of documents.
+    # Therefore, if the number of MOCs is above 10, arbitrarily use a lower chunk size.
+    max_onet_count = max([len(job.get('mapped_moc', [])) for job in jobs])
```
Comment (JLMcLaugh) [inline import_jobs.py]: Imports for large job collections are already slow from the large dictionary and xml objects being created. Let's avoid making another large list. Use a loop and break if we're over the max_one_count.

File: import_jobs.py [hunk 2]
```diff
@@ -613,6 +613,16 @@ def add_jobs(jobs, upload_chunk_size=1024):
     """
     conn = Solr(settings.HAYSTACK_CONNECTIONS['default']['URL'])
     num_jobs = len(jobs)
+    
+    # AT&T Showed that large numbers of MOCs can cause import issues due to the size of documents.
+    # Therefore, if the number of MOCs is above 10, arbitrarily use a lower chunk size.
+    for job in jobs:
+        if len(job.get('mapped_moc', [])) > 100:
```
Comment (JLMcLaugh) [inline import_jobs.py]: Do we want 10 or 100 here?

Comment (almathew) [inline import_jobs.py]: I picked a random Company that had mapped mocs but not as many as ATT. They had ~260 per job.

Comment (tdphillips) [inline import_jobs.py]: I think this is referring to the comment, which implies len(mapped_moc) > 10.

**Pull request merged**